project-generator/src/main/java/com/redhat/ipaas/project/converter/DefaultProjectGenerator.java

```diff
@@ -137,9 +136,11 @@ public DefaultProjectGenerator(ConnectorCatalog connectorCatalog, ProjectGenerat
     }
     */
 
-    private byte[] generateFlowYaml(GenerateProjectRequest request) throws JsonProcessingException {
+    private byte[] generateFlowYaml(Map<String, byte[]> contents, GenerateProjectRequest request) throws JsonProcessingException {
         Flow flow = new Flow();
+        int stepCounter[] = new int[]{0};
         request.getIntegration().getSteps().ifPresent(steps -> {
+            stepCounter[0]++;
```

Comment:
Still don't get it. Isn't it supposed that the counter is _within_ the loop over all steps ? afais (sorry, its a bit messed up in the diffs view, probably need to checkout) the counter is increased only once,  so always 1 used in the conditional below. But I have to check out the code to verify this ...